kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/Batch.java

```diff
@@ -170,4 +170,44 @@ public boolean equals(Object o) {
   public int hashCode() {
     return Objects.hash(getId());
   }
+
+  @Override
+  public String toString() {
+    return "Batch{"
+        + "id='"
+        + id
+        + '\''
+        + ", user='"
+        + user
+        + '\''
+        + ", batchType='"
+        + batchType
+        + '\''
+        + ", name='"
+        + name
+        + '\''
+        + ", appId='"
+        + appId
+        + '\''
+        + ", appUrl='"
+        + appUrl
+        + '\''
+        + ", appState='"
+        + appState
+        + '\''
+        + ", appDiagnostic='"
+        + appDiagnostic
+        + '\''
+        + ", kyuubiInstance='"
+        + kyuubiInstance
+        + '\''
+        + ", state='"
+        + state
+        + '\''
+        + ", createTime="
+        + createTime
+        + ", endTime="
+        + endTime
+        + '}';
```

Comment:
is it generated by idea ? why looks so complex..

Reply:
yes, all the toString methods are generated by IDEA.

And then execute `dev/reformat` to format the code style.

Reply:
magic style..

Reply:
@pan3793  suggested to use ReflectionToStringBuilder, let me have a try.

Reply:
<img width="1467" alt="image" src="https://user-images.githubusercontent.com/6757692/180915779-84fab3c6-d845-4e28-879b-503b257eb33e.png">